examples/todos/src/actions/index.js

```diff
@@ -1,22 +1,22 @@
 let nextTodoId = 0
-export const addTodo = (text) => ({
+export const addTodo = text => ({
   type: 'ADD_TODO',
   id: nextTodoId++,
   text
 })
 
-export const setVisibilityFilter = (filter) => ({
+export const setVisibilityFilter = filter => ({
   type: 'SET_VISIBILITY_FILTER',
   filter
 })
 
-export const toggleTodo = (id) => ({
+export const toggleTodo = id => ({
   type: 'TOGGLE_TODO',
   id
 })
 
 export const VisibilityFilters = {
-  SHOW_ALL: 'SHOW_ALL',
-  SHOW_COMPLETED: 'SHOW_COMPLETED',
-  SHOW_ACTIVE: 'SHOW_ACTIVE'
+ SHOW_ALL: 'SHOW_ALL',
+ SHOW_COMPLETED: 'SHOW_COMPLETED',
+ SHOW_ACTIVE: 'SHOW_ACTIVE'
```

Comment:
This should still be two spaces. Are you running Prettier locally? We've added it to the library code, but not the examples yet. It may be a good idea to introduce it now, since this is already a pretty notable refactor.

Reply:
Hmm. Not sure how that happened, I'm not using Prettier locally either. Maybe consider creating a issue about it separately and it might get taken up in another pull request. Fixed it for now.